ovos_stt_plugin_citrinet/__init__.py

```
@@ -1,25 +1,36 @@
-        LOG.info(f"preloading model: {Model.default_models[lang]}")
-        self.load_model(lang)
+        if self.model_id and os.path.exists(self.model_id):
+            LOG.info(f"loading local model: {self.model_id}")
+            self.models[self.lang] = Model(lang=lang, model_folder=self.model_id)
+        elif self.model_id:
+            LOG.info(f"loading hf model: {self.model_id}")
+            self.models[self.lang] = Model(lang=lang, hf_model=self.model_id)
+        else:
+            LOG.info(f"loading default lang model: {lang}")
+            if lang not in self.available_languages:
+                raise ValueError(f"unsupported language '{lang}', must be one of {self.available_languages}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)